lib/jellyfish_web/api_spec/component/file.ex

```diff
@@ -18,6 +18,12 @@ defmodule JellyfishWeb.ApiSpec.Component.File do
           type: :string,
           description:
             "Relative path to track file. Must be either OPUS encapsulated in Ogg or raw h264"
+        },
+        framerate: %Schema{
+          type: :integer,
+          description: "Framerate of video in a file. It is only valid for video track",
+          nullable: true,
```

Comment:
I think `framerate` should only be nullable in `Options`. We expect Jellyfish to set it to some value.
```suggestion
```